alacritty/src/renderer/builtin_font.rs

```diff
@@ -0,0 +1,700 @@
+                let buffer = canvas.buffer_mut();
+                for y in 1..height {
+                    let left = (y - 1) * width;
+                    let right = y * width - 1;
+                    if extra_offset != 0 {
+                        buffer[right] = buffer[left];
+                    }
+                    for offset in 0..center {
+                        buffer.swap(left + offset, right - offset - extra_offset);
+                    }
+                }
+            }
+            // Mirror `Y` axis.
+            if character == '\u{256d}' || character == '\u{256e}' {
+                let center = canvas.y_center() as usize;
+                let extra_offset = if height % 2 == 0 { 1 } else { 0 };
+                let buffer = canvas.buffer_mut();
+                if extra_offset != 0 {
+                    let bottom_row = (height - 1) * width;
+                    for index in 0..width {
+                        buffer[bottom_row + index] = buffer[index];
+                    }
+                }
+                for offset in 1..=center {
+                    let top_row = (offset - 1) * width;
+                    let bottom_row = (height - offset - extra_offset) * width;
+                    for index in 0..width {
+                        buffer.swap(top_row + index, bottom_row + index);
+                    }
+                }
+            }
+        },
+        // Diagonals: '╱', '╲', '╳'.
+        '\u{2571}' | '\u{2572}' | '\u{2573}' => {
+            let width = width as f32;
+            let height = height as f32;
+            for stroke_size in 0..=stroke_size {
+                let stroke_size = stroke_size as f32 / 2.;
+                if character == '\u{2571}' || character == '\u{2573}' {
+                    canvas.draw_line(stroke_size, height - 1., width - 1., stroke_size);
+                    canvas.draw_line(0., height - 1. - stroke_size, width - 1. - stroke_size, 0.);
+                }
+                if character == '\u{2572}' || character == '\u{2573}' {
+                    canvas.draw_line(stroke_size, 0., width - 1., height - 1. - stroke_size);
+                    canvas.draw_line(0., stroke_size, width - 1. - stroke_size, height - 1.);
+                }
+            }
+        },
+        // Parts of full block: '▀', '▁', '▂', '▃', '▄', '▅', '▆', '▇', '▔', '▉', '▊', '▋', '▌',
+        // '▍', '▎', '▏', '▐', '▕'.
+        '\u{2580}'..='\u{2587}' | '\u{2589}'..='\u{2590}' | '\u{2594}' | '\u{2595}' => {
+            let width = width as f32;
+            let height = height as f32;
+            let rect_width = match character {
+                '\u{2589}' => width * 7. / 8.,
+                '\u{258a}' => width * 6. / 8.,
+                '\u{258b}' => width * 5. / 8.,
+                '\u{258c}' | '\u{2590}' => width * 4. / 8.,
+                '\u{258d}' => width * 3. / 8.,
+                '\u{258e}' => width * 2. / 8.,
+                '\u{258f}' | '\u{2595}' => width / 8.,
```

Comment:
```suggestion
                '\u{258f}' | '\u{2595}' => width * 1. / 8.,
```

Would maybe even like to see the `|` grouping removed here, just to have **everything** orderly and under each other making it clear what's going on.